tests/src/test/scala/common/Wsk.scala

```diff
@@ -1003,11 +1003,12 @@ trait RunWskCmd extends Matchers {
             verbose: Boolean = false,
             env: Map[String, String] = Map("WSK_CONFIG_FILE" -> ""),
             workingDir: File = new File("."),
+            stdinFile: File = null,
```

Comment:
With all other parameters are not `Option` parameters, would introducing an `Option` parameter be confusing?